packages/disputer/test/Disputer.js

```diff
@@ -553,14 +276,286 @@ contract("Disputer.js", function(accounts) {
             empProps,
             disputerConfig
           });
+        });
+
+        versionedIt([{ contractType: "any", contractVersion: "any" }])(
```

Comment:
here you can see the syntax, same as the liquidator, for running multiple versions against the unit tests.

Reply:
What does the `It` in `versionedIt` stand for?

Reply:
it's the mocha and chai `it` assertion, as with normal js unit tests. same pattern as in the docs, as can be seen [here](https://mochajs.org/#getting-started) except now you can define an array of contract versions to run the tests against.